### ISSUE TITLE
Item 7170 (classes 1;3;20 at least) frozen since 22/06/2024

### ISSUE THREAD

**Issue by NicolasPyIGN** [osm-fr/osmose-backend#2363]
Hi,
We consume the false positive of item 7170 classes 1;3;20 and we discovered the update timestamp (last run of analyzer we presume) is frozen at 22 june 2024 22:29.
Do you confirm that ? if yes, is it nominal or something is broken backend side ?

**Comment by frodrigo** [comment]
Indeed, this classes are not updated for a while. The update come from an external tool by @cquest .

You also maybe interested y the new class 13

https://osmose.openstreetmap.fr/fr/issues/open?item=7170

**Comment by NicolasPyIGN** [comment]
In
https://wiki.openstreetmap.org/wiki/FR:Osmose/issues, Classe 13 "voirie cadastre sans route à 20 m" 
also
../analysers/analyser_merge_road_FR.py:32 link to IGNF:BDTopo, not french cadaster institute

If I understand clearly, item 7170 class 13 do the comparison between OSM and official road network. If so, a false positive on that class indicate a potential error in the official road network. Is it correct ?

**Comment by frodrigo** [comment]
13 compare OSM and IGN BDTOPO

https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_road_FR.py#L39

**Comment by NicolasPyIGN** [comment]
We looked at https://osmose.openstreetmap.fr/fr/issues/open?item=7170 class 13.
Until now, we used country=france and source=14708 ; we understood that countries were independant, and non hierarchical.

Looking at https://osmose.openstreetmap.fr/fr/issues/open?item=7170&source=&class=13&username=&bbox=, we discovered there's now many country and source.

What are we missing ? How works that new item7170+class13 ?

**Comment by frodrigo** [comment]
item7170+class13 is a standard case. Using only one source for the whole country a particular case.

> What are we missing ?

What are you expected ?

**Comment by NicolasPyIGN** [comment]
Until now we looked at:
_false-positive
and
__item7170 source=14708 class 1 country=france*
__item7170 source=14708 class 3 country=france*
__item7170 source=14708 class 4 country=france*
__item7170 source=14708 class 20 country=france*

We expected having to add
false-positive
item7170 
class13
country=france*
attached to only one unique source

Looking at 
https://osmose.openstreetmap.fr/api/0.3/issues.json?limit=10000&country=france*&full=true&status=false&start_date=2024-08-17&end_date=2024-10-17&useDevItem=false&class=13&item=7170
show at least 2 sources: 428617 and 428654. 

Why many sources? are they perennial ?

**Comment by frodrigo** [comment]
Better no use the source parameter.

1 source = (1 analyser, 1 area)

**Comment by NicolasPyIGN** [comment]
so only
_false positive
__item7170 class 1 country=france*
__item7170 class 3 country=france*
__item7170 class 4 country=france*
__item7170 class 20 country=france*
__item7170 class 13 country=france*
does the job ?
